### ISSUE TITLE
hu-HU: langmetadata.xml contains incorrect firstDay/weekEnd entries

### ISSUE THREAD

**Issue by DelPoint** [joomla/core-translations#78]
Hi,
I've just noticed that the **langmetadata.xml** files for Hungarian translation contains incorrect entries between tags **<firstDay>** and **<weekEnd>**.

This is the current - incorrect - state:
```
<firstDay>0</firstDay>
<weekEnd>0,6</weekEnd>
```

And this should be the correct one IMHO:
```
<firstDay>1</firstDay>
<weekEnd>6,7</weekEnd>
```
Because of the wrong entries, calendars will show non-native ordering of week days.

What am I supposed to do? Create a pr, or something else? I've seen some comments regarding that _langmetadata.xml_ is handled inside Crowdin, but I'm not sure what this actually means in this case.

**Comment by infograf768** [comment]
Send Honzak a private message in the forum https://forum.joomla.org/ucp.php?i=pm&mode=compose&u=702152 or send a mail szerk@joomlacms.hu

**Comment by DelPoint** [comment]
Thx, for the reply/help. 
Sent PM to Honzak. As soon as I have a positive reply from him I'll close this issue here.

**Comment by conconnl** [comment]
Honzakr is the proofreader for Czech not Hungarian.
The coordinator for Hungarian is: 
Balazs Csontos
Forum profile: https://forum.joomla.org/memberlist.php?mode=viewprofile&u=195161
Mail address (indeed): szerk@joomlacms.hu

You can also find the info here: https://downloads.joomla.org/language-packs/translations-joomla4

**Comment by infograf768** [comment]
I stand corrected. Tks

**Comment by csbalazs** [comment]
The issue is fixed in the 4.0.5 language pack.

**Comment by conconnl** [comment]
Thank you.
